app/controllers/webhooks_controller.rb

```diff
@@ -36,4 +36,27 @@ def gocardless
 
     head(:ok)
   end
+
+  def adyen
+    signature = adyen_params.dig("additionalData", "hmacSignature")
```

Comment:
Lint/UselessAssignment: Useless assignment to variable - `signature`. (https://rubystyle.guide#underscore-unused-vars)